PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+PULL REQUEST TEMPLATE
+======================
+
+
+### Description
+> Please include a summary of the change, relevant motivation and context. If possible and applicable, include before and after screenshots and a URL where the changes can be seen.
+    
+Check any of the following which apply:
+
+* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
+* [ ]  This change requires a documentation update
+    * [ ]  I have made corresponding changes to the documentation
+* [ ]  This change requires new dependencies (please list)
+
+### Testing
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+* Test A
+* Test B
+
+### Checklist
+* [ ]  I have commented my code, particularly in hard-to-understand areas
+* [ ]  My changes generate no new warnings
+
```

Comment:
` * [ ] I have not used code from external sources without attribution`

Reply:
`* [ ] I have considered accessibility in my implementation

Reply:
`* [ ] There are no remaining debug statements (print, console.log, ...)`

Reply:
done